src/shared/services/activity/activity.service.ts

```diff
@@ -13,11 +14,17 @@ export class ActivityService {
   static get ACTIVITY_VIEW() { return 'view'; }
   static get ACTIVITY_EDIT() { return 'edit'; }
 
-  private userAuthorised;
+  private userAuthorised: boolean = undefined;
```

Comment:
Ohh an undefined boolean. Not seen one of these in a while!

Reply:
It was an untyped variable... so I gave it a type... but then the logic required that it be `undefined` before proceeding. I suppose I could have used some magic ~beans~ strings instead... 😂

Reply:
Besides, it's usually considered a good thing to be first. Do I get a 🥇 now?